src/content/blog/astro-460.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: "Astro 4.6"
+description: "Astro 4.6 is here! This release includes a new manual routing strategy for internationalization, the ability to move the dev toolbar, experimental support for CSRF protection and more."
+publishDate: "April 11, 2024"
+authors:
+  - erika
+  - ema
+  - matthew
+  - nate
+  - bjorn
+coverImage: "/src/content/blog/_images/astro-460/post-header-4.6.webp"
+socialImage: "/src/content/blog/_images/astro-460/og-image-4.6.webp"
+lang: "en"
+---
+
+import BlogSummary from "/src/components/BlogSummary.astro";
+
+<BlogSummary>
+  After a short break post-launch week, Astro 4.6 is now here and we're back to our regular schedule! This release includes a new manual routing strategy for internationalization, experimental support for CSRF protection, a new feature for the dev toolbar and more.
+</BlogSummary>
+
+Full release highlights include:
+
+- [**Manual routing strategy for internationalization**](#manual-routing-strategy-for-internationalization)
+- [**Ability to move the dev toolbar**](#ability-to-move-the-dev-toolbar)
+- [**Experimental: Support for CSRF protection**](#experimental-support-for-csrf-protection)
+- [**Cookies improvements**](#cookies-improvements)
```

Comment:
```suggestion
- [**Cookies improvements**](#cookies-improvements)
- [**Deprecated support for older versions of Node](#deprecated-support-for-older-versions-of-node)
```
Agree with Matthew that this could go last. (Which means also moving the paragraph to the end)